website/static/css/code-block-buttons.css

```diff
@@ -0,0 +1,55 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+pre {
+  position: relative;
+}
+
+pre .copyCodeButton {
+  background: none;
+  border: none;
+  color: #9caeff;
+  cursor: pointer;
+  position: absolute;
+  right: 0px;
+  top: 4px;
+  z-index: 1;
+}
+
+pre .copyCodeButton:focus {
+  outline: none;
+}
+
+pre .copyCodeButton::-moz-focus-inner {
+  border: none;
+}
+
+pre .copyCodeButton:hover {
+  text-decoration: none;
```

Comment:
Might be even better if the color of the button changes (to black) when hovering over, similar to the edit button.

Reply:
I was trying to get it so that the cursor changes to a pointer instead of the button color changing. Not sure if the CSS that I currently have is optimal though.

Reply:
Okay that's fine then.